config/certificate/kustomizeconfig.yaml

```diff
@@ -14,3 +14,9 @@ varReference:
 - kind: Certificate
   group: cert-manager.io
   path: spec/dnsNames
+
+namePrefix:
+- kind: Certificate
+  group: cert-manager.io
+  name: serving-cert
+  path: spec/secretName
```

Comment:
This one is already prefixed by kustomize because it is a applied by kustomize:
```
ybettan kernel-module-management (e2e-hub) $ oc get certificate --all-namespaces
NAMESPACE             NAME                            READY   SECRET                    AGE
kmm-operator-system   kmm-operator-hub-serving-cert   True    webhook-hub-server-cert   2d
kmm-operator-system   kmm-operator-serving-cert       True    webhook-server-cert       2d
```

Reply:
The PR prefixes the Secret name generated by cert-manager as configured in the `Certificate` resource.